person-create.component.js

```
@@ -1,13 +1,147 @@
+    city: '',
+    stateProvidenceDistrict: '',
+    postalCode: '',
+    country: '',
+  };
+}
+typeChangeonChange1(0);
+
+console.timeEnd("code refactor 1");
+console.time("code refactor 2");
+
+function typeChangeonChange1(index) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'typeChangeonChange1'. Consider to delete it or rename it.

'typeChangeonChange1' is defined here:


(lint/suspicious/noRedeclare)